analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/LocksShouldBeReleased.Mutex.NetFx.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Security.AccessControl;
+using System.Threading;
+
+// .NET Framework 4.8 APIs https://docs.microsoft.com/en-us/dotnet/api/system.threading.mutex.-ctor?view=netframework-4.8
+
+// In .NET 6, the constructors/methods with MutexSecurity and MutexRights are in the MutexAcl class inside the Microsoft.Windows.Compatibility NuGet.
```

Comment:
here is why I added an IT

Reply:
It would be nice to have the reason in the IT too. To make sure somebody does not remove it.